superset/views/core.py

```diff
@@ -2481,14 +2481,7 @@ def results(self, key):
             return json_error_response(get_datasource_access_error_msg(
                 '{}'.format(rejected_tables)))
 
-        payload = utils.zlib_decompress_to_string(blob)
-        display_limit = app.config.get('SQL_MAX_ROW', None)
-        if display_limit:
-            payload_json = json.loads(payload)
-            payload_json['data'] = payload_json['data'][:display_limit]
-        return json_success(
-            json.dumps(
-                payload_json, default=utils.json_iso_dttm_ser, ignore_nan=True))
+        return json_success(utils.zlib_decompress_to_string(blob))
```

Comment:
Why are the `default` and `ignore_nan` arguments no longer valid?

Reply:
When the blob is written in celery, it is already set with thos variables. 
https://github.com/apache/incubator-superset/blob/master/superset/sql_lab.py#L229

Reply:
I added ignore-nan

Reply:
Turns out `ignore_nan` is  breaking this method. I believe it's a simplejson arg only